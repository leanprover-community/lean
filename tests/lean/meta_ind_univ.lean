-- should fail
inductive foo : Type → Type
| mk : Π (α β : Type), (β → α) → foo α

-- should be fine
meta inductive foom : Type → Type
| mk : Π (α β : Type), (β → α) → foom α

