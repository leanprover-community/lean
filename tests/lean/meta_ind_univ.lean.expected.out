meta_ind_univ.lean:2:0: error: universe level of type_of(arg #1) of 'foo.mk' is too big for the corresponding inductive datatype
